infiniband/assets/configuration/spec.yaml

```diff
@@ -10,14 +10,14 @@ files:
         options:
           - name: infiniband_path
             description: |
-              The path to the infiniband directory.
+              The path to the InfiniBand directory.
```

Comment:
I notice that in the README file, references to the directory were left with the lowercase, whereas references to the integration itself were capitalized. We might want to consider reverting the capitalization changes in the file, if we want to make it clear that the directory is called `infiniband` and not `InfiniBand`.

Reply:
Yeah make sense to me. I think I've addressed it now.